src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -35,7 +35,7 @@ public OverseasEntitiesController(OverseasEntitiesService overseasEntitiesServic
         this.overseasEntitiesService = overseasEntitiesService;
     }
 
-    @PostMapping("/")
+    @PostMapping
```

Comment:
are you able to update the schema.json spec as well? i think that has trailing / on it

Reply:
Oh yes good shout.